src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -725,3 +725,35 @@ void WorldSession::HandleSummonResponseOpcode(WorldPacket& recvData)
     _player->SetSummonAsSpectator(false);
     _player->SummonIfPossible(agree, summoner_guid);
 }
+
+void WorldSession::HandleMoveTimeSkippedOpcode(WorldPacket& recvData)
+{
+    sLog->outString("WORLD: Received CMSG_MOVE_TIME_SKIPPED");
```

Comment:
I think this should be changed as follows. At least it is handled this way in other debug messages:
```suggestion
#if defined(ENABLE_EXTRAS) && defined(ENABLE_EXTRA_LOGS)
    sLog->outDebug(LOG_FILTER_NETWORKIO, "WORLD: Recvd CMSG_MOVE_TIME_SKIPPED");
#endif
```

Reply:
yes thank you for example stoabrogga!